nautobot/extras/tests/test_models.py

```diff
@@ -1257,6 +1260,23 @@ def test_log_entry_creation(self):
         self.assertEqual(log_object.log_level, log.log_level)
         self.assertEqual(log_object.grouping, log.grouping)
 
+    def test_to_csv(self):
+        """Check that `to_csv` returns the correct data from the JobLogEntry model."""
+        expected_data = (datetime.datetime(2020, 1, 26, 15, 37, 36), "run", "success", "", "Django Test")
```

Comment:
Shouldn't the CSV data contain the string representation of the timestamp, rather than a `datetime` object?

Reply:
Added additional test.